sample/app/src/main/java/com/example/activitystarter/SavingTestActivity.java

```diff
@@ -1,15 +1,16 @@
 package com.example.activitystarter;
 
-import android.support.v7.app.AppCompatActivity;
 import android.os.Bundle;
 import android.view.View;
 import android.widget.Button;
 import android.widget.TextView;
 
 import activitystarter.ActivityStarter;
 import activitystarter.Arg;
+import androidx.appcompat.app.AppCompatActivity;
 
-public class SavingTestActivity extends AppCompatActivity {
+public class SavingTestActivity extends AppCompatActivity
+{
```

Comment:
It doesn't look like Kotlin style
https://kotlinlang.org/docs/reference/coding-conventions.html